platform/platform.go

```diff
@@ -158,6 +161,9 @@ type Options struct {
 	// When specified additional files & units will be automatically generated
 	// inside of RenderUserData
 	OSContainer string
+
+	// Board is the board used by the image
+	Board string
```

Comment:
I don't follow; when this variable is added here, how should it relate to the existing `Board` variable used in `../platform/api/packet/api.go`?
Above it doesn't get set because only `kola.PacketOptions.Board = board` is assigned. I would recommend to unify this: either use only the new `Board` defined here or add `Board` variables to all platform options instead of here.

Reply:
yeah, it's a bit confusing - basically we are in this use-case for providers != qemu & packet:
```go
type A struct {
	*B
}

type B struct {
	board string
}

a := A{&B{}}
fmt.Printf("%p\n", &a.board) // 0xc00008e000
fmt.Printf("%p\n", &a.B.board) // 0xc00008e000
```

So for `qemu` and `packet` we can simply remove the custom `Board` for each in order to rely on the unique `Board` defined in `platform.Options`.

Reply:
Thanks, looks good now - previously I think the wrong variable was used for Packet/QEMU in the SELinux test, now it's more clear what's going on.